### ISSUE TITLE
Error when open the App in Teams

### ISSUE THREAD

**Issue by dborchers-gc** [OfficeDev/microsoft-teams-apps-greatideas#18]
When i open the app in Teams the App Ideas Tab shows me: **The resource you are looking for has been removed, had its name changed, or is temporarily unavailable.**

Wht i am doing wrong? Ive setup the apps as described in the deployment guide

Any Idea?

**Comment by BraunJonathan** [comment]
The app iself has a deployment problem right now, it is caused by a dependency issue...
You can have a look at this Pull Request : https://github.com/OfficeDev/microsoft-teams-apps-greatideas/pull/16
With this changes i managed to deploy and run the ap in Teams (but i'm not able to create new categories as stated in the issue i opened last friday...)

**Comment by dborchers-gc** [comment]
ah okay - is there a way for me to fix that manually in my deployment?

**Comment by BraunJonathan** [comment]
you can change the github repository and branch used for the deployment and use the one which is at the origin of the pull request.
You can also fork the project, fix the issue yourself and use your own github repository to deploy the app...

**Comment by dborchers-gc** [comment]
Ive redeploy the App from your repro but i get teh same error. Do you have another diea?

**Comment by BraunJonathan** [comment]
Did you check which error you got in the deployment section of your appService?

**Comment by dborchers-gc** [comment]
Ive got an Conflict Error on the Web step but after a redeploy it was completed
![2020-09-14 11_04_57-Microsoft Template-20200911125648 - Microsoft Azure und 20 weitere Seiten - DB O](https://user-images.githubusercontent.com/63800439/93066452-50c8c780-f67a-11ea-83ae-783c8915301d.png)

**Comment by BraunJonathan** [comment]
Yes, it was the same for me but if you check your deployments in the appservice, you'll probably see an error there...
To make it "work" (quoted as i still have a problem with categories...) i deleted the apService and redployed the whole thing from the other github repository...

**Comment by laracloud123** [comment]
@dborchers-gc did you fix this issue? I still get the failure message: The resource you are looking for has been removed, had its name changed, or is temporarily unavailable. thanks :)
